src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+  const getSum = (num1) => {
+    return function(num2) {
+      counter++;
+
+      if (counter >= 4 && counter % 2 === 0) {
```

Comment:
suggestion
```suggestion
      if (counter > 3 && counter % 2 === 0) {
```
it looks more obvious